cms/envs/production.py

```diff
@@ -350,6 +350,8 @@ def get_env_setting(setting):
 
 USER_TASKS_ARTIFACT_STORAGE = COURSE_IMPORT_EXPORT_STORAGE
 
+COURSE_METADATA_EXPORT_STORAGE = 'cms.djangoapps.export_course_metadata.storage.CourseMetadataExportS3Storage'
```

Comment:
not sure which environment applies for staging

Reply:
tried to do the same here - linked the edx-internal PR.
should there be some sort of additional setting in common.py/production.py/edx-internal that enables the feature (in addition to the waffle flag) so open source can disable the feature once the waffle flag is removed?  
not sure when we decide if that's necessary (though exporting a file to their production machines seems unnecessary?)

Reply:
An open edx installation would just not set `COURSE_METADATA_EXPORT_BUCKET` yeah? Then the S3 path isn't used (we'd still export to a file, but that seems harmless?)

Reply:
i think it might be multiple files, but yeah i suppose that's harmless since it wouldn't actually be consuming a lot of space

Reply:
Still though, maybe you're right that it's better to have like a feature flag (not a waffle flag) for this? I dunno. I don't think it's *bad* to leave all these content files around... but...  

Maybe you should just disable the feature if the bucket isn't sent, instead of falling back to files?